My project/MyfirstgameinC#1/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿Random random = new Random();
+
+Console.WriteLine("Hello! Would you like to play my game?");
+var player1 = Console.ReadLine();
+
+if (player1 == "Yes")
+{
+    var bot = random.Next(50);
+    Console.Clear();
+    int attempts = 0;
+    while (true)
```

Comment:
Bad practise, I see that you have added 'break' but it's better to avoid such cases